src/logdb/test/logdb_tests.c

```diff
@@ -120,7 +120,7 @@ void test_logdb(logdb_log_db* (*new_func)())
     u_assert_int_eq(memcmp(value_test->str, value->str, value->len), 0);
 
     value_test = logdb_find(db, key2);
-    u_assert_int_eq((int)value_test, 0); /* should be null */
+    u_assert_int_eq(!value_test, 1); /* should be null */
```

Comment:
Ah. I see. This tests a pointer for not being null (I though it tests its 32bit value).
I think today I would write this `u_assert_int_eq(value_test == NULL, false);` or better: adding a `u_assert_not_null(void *)` to `utest.h`.